### PR TITLE
OCM-8052 | test: Automated id:65798 can customize ingress at install

### PR DESCRIPTION
**Pull request by jerichokeyne**
Successful output:
```
$ TEST_PROFILE=rosa-advanced CLUSTER_ID=2bdaig3783olno64f3h9pbpah1hvnagc ginkgo run -focus 65798 tests/e2e/
Running Suite: e2e tests suite - /home/jericho/work/repos/rosa/tests/e2e
========================================================================
Random Seed: 1716409068
                                                                                                                                                                                             
Will run 1 of 79 specs
SSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
                                                                                                                                                                                             
Ran 1 of 79 Specs in 2.065 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 78 Skipped
PASS
                                                                                                                                                                                             
Ginkgo ran 1 suite in 3.009243034s
Test Suite Passed
```

Failed output:
```
❯ TEST_PROFILE=rosa-advanced CLUSTER_ID=2bdae83sg599t6mvhs7f32k99kmfmnhr ginkgo run -focus 65798 tests/e2e/
Running Suite: e2e tests suite - /home/jericho/work/repos/rosa/tests/e2e
========================================================================
Random Seed: 1716408847
                                                                                                                                                                                             
Will run 1 of 79 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
------------------------------
• [FAILED] [3.290 seconds]
Edit default ingress [It] can customize ingress controller at install - [id:65798] [day2, feature-ingress, High, day1-post, NonHCPCluster]
/home/jericho/work/repos/rosa/tests/e2e/test_rosacli_ingress.go:183
                                                                                                                                                                                             
  Timeline >>
  STEP: Get the cluster @ 05/22/24 16:14:08.887
  STEP: Init the client @ 05/22/24 16:14:08.887
  STEP: Check cluster is hosted @ 05/22/24 16:14:08.887
  2024-05-22T16:14:08-04:00 - INFO : Running command: rosa describe cluster -c 2bdae83sg599t6mvhs7f32k99kmfmnhr --output json
  Got need redacted string from log match regex (arn:aws:[a-z]+:[a-z0-9-]*:)([0-9]{12})(:)
  2024-05-22T16:14:10-04:00 - DEBUG : Get Combining Stdout and Stder is :
  {
    "addons": {
      "items": []
    },
    "api": {
      "listening": "external",
      "url": "https://api.jkeyne-20240521.blq1.s1.devshift.org:6443"
    },
    "aws": {
      "audit_log": {
        "role_arn": ""
      },
      "ec2_metadata_http_tokens": "optional",
      "private_link": false,
      "private_link_configuration": {},
      "sts": {
        "auto_mode": false,
        "enabled": true,
        "instance_iam_roles": {
          "master_role_arn": "arn:aws:iam::*************:role/jkeyne-ControlPlane-Role",
          "worker_role_arn": "arn:aws:iam::*************:role/jkeyne-Worker-Role"
        },
        "managed_policies": false,
        "oidc_endpoint_url": "https://oidc.os1.devshift.org/2bdae83sg599t6mvhs7f32k99kmfmnhr",
        "operator_iam_roles": [
          {
            "id": "",
            "name": "aws-cloud-credentials",
            "namespace": "openshift-machine-api",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-machine-api-aws-cloud-credentials",
            "service_account": ""
          },
          {
            "id": "",
            "name": "cloud-credential-operator-iam-ro-creds",
            "namespace": "openshift-cloud-credential-operator",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-cloud-credential-operator-cloud-c",
            "service_account": ""
          },
          {
            "id": "",
            "name": "installer-cloud-credentials",
            "namespace": "openshift-image-registry",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-image-registry-installer-cloud-cr",
            "service_account": ""
          },
          {
            "id": "",
            "name": "cloud-credentials",
            "namespace": "openshift-ingress-operator",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-ingress-operator-cloud-credential",
            "service_account": ""
          },
          {
            "id": "",
            "name": "ebs-cloud-credentials",
            "namespace": "openshift-cluster-csi-drivers",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-cluster-csi-drivers-ebs-cloud-cre",
            "service_account": ""
          },
          {
            "id": "",
            "name": "cloud-credentials",
            "namespace": "openshift-cloud-network-config-controller",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-cloud-network-config-controller-c",
            "service_account": ""
          }
        ],
        "operator_role_prefix": "jkeyne-20240521-f1a1",
        "role_arn": "arn:aws:iam::*************:role/jkeyne-Installer-Role",
        "support_role_arn": "arn:aws:iam::*************:role/jkeyne-Support-Role"
      },
      "tags": {
        "red-hat-clustertype": "rosa",
        "red-hat-managed": "true"
      }
    },
    "aws_infrastructure_access_role_grants": {
      "items": []
    },
    "billing_model": "standard",
    "byo_oidc": {
      "enabled": false
    },
    "ccs": {
      "disable_scp_checks": false,
      "enabled": true,
      "kind": "CCS"
    },
    "cloud_provider": {
      "href": "/api/clusters_mgmt/v1/cloud_providers/aws",
      "id": "aws",
      "kind": "CloudProviderLink"
    },
    "console": {
      "url": "https://console-openshift-console.apps.jkeyne-20240521.blq1.s1.devshift.org"
    },
    "creation_timestamp": "2024-05-21T19:25:23Z",
    "delete_protection": {
      "enabled": false
    },
    "disable_user_workload_monitoring": false,
    "displayName": "jkeyne-20240521",
    "dns": {
      "base_domain": "blq1.s1.devshift.org"
    },
    "domain_prefix": "jkeyne-20240521",
    "etcd_encryption": false,
    "expiration_timestamp": "2024-05-23T07:25:20Z",
    "external_auth_config": {
      "enabled": false,
      "external_auths": {
        "items": []
      }
    },
    "external_configuration": {
      "labels": {
        "items": []
      },
      "manifests": {
        "items": []
      },
      "syncsets": {
        "items": []
      }
    },
    "external_id": "1132f282-0176-4ce1-9788-7a2dea3b110c",
    "flavour": {
      "href": "/api/clusters_mgmt/v1/flavours/osd-4",
      "id": "osd-4",
      "kind": "FlavourLink"
    },
    "groups": {
      "items": []
    },
    "href": "/api/clusters_mgmt/v1/clusters/2bdae83sg599t6mvhs7f32k99kmfmnhr",
    "hypershift": {
      "enabled": false
    },
    "id": "2bdae83sg599t6mvhs7f32k99kmfmnhr",
    "identity_providers": {
      "items": []
    },
    "inflight_checks": {
      "items": []
    },
    "infra_id": "jkeyne-20240521-g4896",
    "ingresses": {
      "items": []
    },
    "kind": "Cluster",
    "machine_pools": {
      "items": []
    },
    "managed": true,
    "managed_service": {
      "enabled": false
    },
    "multi_az": false,
    "name": "jkeyne-20240521",
    "network": {
      "host_prefix": 23,
      "machine_cidr": "10.0.0.0/16",
      "pod_cidr": "10.128.0.0/14",
      "service_cidr": "172.30.0.0/16",
      "type": "OVNKubernetes"
    },
    "node_drain_grace_period": {
      "unit": "minutes",
      "value": 60
    },
    "nodes": {
      "availability_zones": [
        "us-east-1a"
      ],
      "compute": 2,
      "compute_machine_type": {
        "href": "/api/clusters_mgmt/v1/machine_types/m5.xlarge",
        "id": "m5.xlarge",
        "kind": "MachineTypeLink"
      },
      "compute_root_volume": {
        "aws": {
          "size": 300
        }
      },
      "infra": 2,
      "infra_machine_type": {
        "href": "/api/clusters_mgmt/v1/machine_types/r5.xlarge",
        "id": "r5.xlarge",
        "kind": "MachineTypeLink"
      },
      "master": 3
    },
    "openshift_version": "4.15.12",
    "product": {
      "href": "/api/clusters_mgmt/v1/products/rosa",
      "id": "rosa",
      "kind": "ProductLink"
    },
    "properties": {
      "rosa_cli_version": "1.2.40",
      "rosa_creator_arn": "arn:aws:iam::*************:user/jkeyne"
    },
    "region": {
      "href": "/api/clusters_mgmt/v1/cloud_providers/aws/regions/us-east-1",
      "id": "us-east-1",
      "kind": "CloudRegionLink"
    },
    "state": "ready",
    "status": {
      "configuration_mode": "full",
      "dns_ready": true,
      "kind": "ClusterStatus",
      "limited_support_reason_count": 0,
      "oidc_ready": true,
      "provision_error_code": "",
      "provision_error_message": "",
      "state": "ready"
    },
    "subscription": {
      "href": "/api/accounts_mgmt/v1/subscriptions/2gn3TiilDDdARQw7Rww26saSQql",
      "id": "2gn3TiilDDdARQw7Rww26saSQql",
      "kind": "SubscriptionLink"
    },
    "version": {
      "channel_group": "stable",
      "end_of_life_timestamp": "2025-06-30T00:00:00Z",
      "href": "/api/clusters_mgmt/v1/versions/openshift-v4.15.12",
      "id": "openshift-v4.15.12",
      "kind": "Version",
      "raw_id": "4.15.12"
    }
  }
                                                                                                                                                                                             
  2024-05-22T16:14:10-04:00 - INFO : Loaded cluster profile configuration from origional profile rosa-advanced : {candidate rosa-advanced  us-west-2 latest 0xc00038b780 0xc000c9c210}
  2024-05-22T16:14:10-04:00 - INFO : Loaded cluster profile configuration from origional cluster rosa-advanced : { required r5.xlarge  unmanaged   3 0 54 150 0 false false true true true true true false false true false true false true true true true true false false true true false false}
  2024-05-22T16:14:10-04:00 - INFO : Loaded cluster profile configuration from origional account-roles rosa-advanced : {/test/ arn:aws:iam::aws:policy/AdministratorAccess}
  2024-05-22T16:14:10-04:00 - INFO : Running command: rosa list ingress -c 2bdae83sg599t6mvhs7f32k99kmfmnhr
  2024-05-22T16:14:12-04:00 - DEBUG : Get Combining Stdout and Stder is :
  ID    APPLICATION ROUTER                                 PRIVATE  DEFAULT  ROUTE SELECTORS  LB-TYPE  EXCLUDED NAMESPACE  WILDCARD POLICY      NAMESPACE OWNERSHIP
  e5m2  https://apps.jkeyne-20240521.blq1.s1.devshift.org  no       yes                       nlb                          WildcardsDisallowed  Strict
                                                                                                                                                                                             
  [FAILED] in [It] - /home/jericho/work/repos/rosa/tests/e2e/test_rosacli_ingress.go:209 @ 05/22/24 16:14:12.176
  << Timeline
                                                                                                                                                                                             
  [FAILED] Expected
      <string>:
  to contain substring
      <string>: test-ns1
  In [It] at: /home/jericho/work/repos/rosa/tests/e2e/test_rosacli_ingress.go:209 @ 05/22/24 16:14:12.176
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
                                                                                                                                                                                             
Summarizing 1 Failure:
  [FAIL] Edit default ingress [It] can customize ingress controller at install - [id:65798] [day2, feature-ingress, High, day1-post, NonHCPCluster]
  /home/jericho/work/repos/rosa/tests/e2e/test_rosacli_ingress.go:209
                                                                                                                                                                                             
Ran 1 of 79 Specs in 3.292 seconds
FAIL! -- 0 Passed | 1 Failed | 0 Pending | 78 Skipped
--- FAIL: TestROSACLIProvider (3.29s)
FAIL
                                                                                                                                                                                             
Ginkgo ran 1 suite in 4.238905069s
                                                                                                                                                                                             
Test Suite Failed
```

Output when missing `TEST_PROFILE`:
```
$ CLUSTER_ID=2bdae83sg599t6mvhs7f32k99kmfmnhr ginkgo run -focus 65798 tests/e2e/
Running Suite: e2e tests suite - /home/jericho/work/repos/rosa/tests/e2e
========================================================================
Random Seed: 1716408799
                                                                                                                                                                                             
Will run 1 of 79 specs
SSSSSSS
------------------------------
• [PANICKED] [1.576 seconds]
Edit default ingress [It] can customize ingress controller at install - [id:65798] [day2, feature-ingress, High, day1-post, NonHCPCluster]
/home/jericho/work/repos/rosa/tests/e2e/test_rosacli_ingress.go:183
                                                                                                                                                                                             
  Timeline >>
  STEP: Get the cluster @ 05/22/24 16:13:20.426
  STEP: Init the client @ 05/22/24 16:13:20.426
  STEP: Check cluster is hosted @ 05/22/24 16:13:20.426
  2024-05-22T16:13:20-04:00 - INFO : Running command: rosa describe cluster -c 2bdae83sg599t6mvhs7f32k99kmfmnhr --output json
  Got need redacted string from log match regex (arn:aws:[a-z]+:[a-z0-9-]*:)([0-9]{12})(:)
  2024-05-22T16:13:22-04:00 - DEBUG : Get Combining Stdout and Stder is :
  {
    "addons": {
      "items": []
    },
    "api": {
      "listening": "external",
      "url": "https://api.jkeyne-20240521.blq1.s1.devshift.org:6443"
    },
    "aws": {
      "audit_log": {
        "role_arn": ""
      },
      "ec2_metadata_http_tokens": "optional",
      "private_link": false,
      "private_link_configuration": {},
      "sts": {
        "auto_mode": false,
        "enabled": true,
        "instance_iam_roles": {
          "master_role_arn": "arn:aws:iam::*************:role/jkeyne-ControlPlane-Role",
          "worker_role_arn": "arn:aws:iam::*************:role/jkeyne-Worker-Role"
        },
        "managed_policies": false,
        "oidc_endpoint_url": "https://oidc.os1.devshift.org/2bdae83sg599t6mvhs7f32k99kmfmnhr",
        "operator_iam_roles": [
          {
            "id": "",
            "name": "aws-cloud-credentials",
            "namespace": "openshift-machine-api",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-machine-api-aws-cloud-credentials",
            "service_account": ""
          },
          {
            "id": "",
            "name": "cloud-credential-operator-iam-ro-creds",
            "namespace": "openshift-cloud-credential-operator",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-cloud-credential-operator-cloud-c",
            "service_account": ""
          },
          {
            "id": "",
            "name": "installer-cloud-credentials",
            "namespace": "openshift-image-registry",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-image-registry-installer-cloud-cr",
            "service_account": ""
          },
          {
            "id": "",
            "name": "cloud-credentials",
            "namespace": "openshift-ingress-operator",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-ingress-operator-cloud-credential",
            "service_account": ""
          },
          {
            "id": "",
            "name": "ebs-cloud-credentials",
            "namespace": "openshift-cluster-csi-drivers",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-cluster-csi-drivers-ebs-cloud-cre",
            "service_account": ""
          },
          {
            "id": "",
            "name": "cloud-credentials",
            "namespace": "openshift-cloud-network-config-controller",
            "role_arn": "arn:aws:iam::*************:role/jkeyne-20240521-f1a1-openshift-cloud-network-config-controller-c",
            "service_account": ""
          }
        ],
        "operator_role_prefix": "jkeyne-20240521-f1a1",
        "role_arn": "arn:aws:iam::*************:role/jkeyne-Installer-Role",
        "support_role_arn": "arn:aws:iam::*************:role/jkeyne-Support-Role"
      },
      "tags": {
        "red-hat-clustertype": "rosa",
        "red-hat-managed": "true"
      }
    },
    "aws_infrastructure_access_role_grants": {
      "items": []
    },
    "billing_model": "standard",
    "byo_oidc": {
      "enabled": false
    },
    "ccs": {
      "disable_scp_checks": false,
      "enabled": true,
      "kind": "CCS"
    },
    "cloud_provider": {
      "href": "/api/clusters_mgmt/v1/cloud_providers/aws",
      "id": "aws",
      "kind": "CloudProviderLink"
    },
    "console": {
      "url": "https://console-openshift-console.apps.jkeyne-20240521.blq1.s1.devshift.org"
    },
    "creation_timestamp": "2024-05-21T19:25:23Z",
    "delete_protection": {
      "enabled": false
    },
    "disable_user_workload_monitoring": false,
    "displayName": "jkeyne-20240521",
    "dns": {
      "base_domain": "blq1.s1.devshift.org"
    },
    "domain_prefix": "jkeyne-20240521",
    "etcd_encryption": false,
    "expiration_timestamp": "2024-05-23T07:25:20Z",
    "external_auth_config": {
      "enabled": false,
      "external_auths": {
        "items": []
      }
    },
    "external_configuration": {
      "labels": {
        "items": []
      },
      "manifests": {
        "items": []
      },
      "syncsets": {
        "items": []
      }
    },
    "external_id": "1132f282-0176-4ce1-9788-7a2dea3b110c",
    "flavour": {
      "href": "/api/clusters_mgmt/v1/flavours/osd-4",
      "id": "osd-4",
      "kind": "FlavourLink"
    },
    "groups": {
      "items": []
    },
    "href": "/api/clusters_mgmt/v1/clusters/2bdae83sg599t6mvhs7f32k99kmfmnhr",
    "hypershift": {
      "enabled": false
    },
    "id": "2bdae83sg599t6mvhs7f32k99kmfmnhr",
    "identity_providers": {
      "items": []
    },
    "inflight_checks": {
      "items": []
    },
    "infra_id": "jkeyne-20240521-g4896",
    "ingresses": {
      "items": []
    },
    "kind": "Cluster",
    "machine_pools": {
      "items": []
    },
    "managed": true,
    "managed_service": {
      "enabled": false
    },
    "multi_az": false,
    "name": "jkeyne-20240521",
    "network": {
      "host_prefix": 23,
      "machine_cidr": "10.0.0.0/16",
      "pod_cidr": "10.128.0.0/14",
      "service_cidr": "172.30.0.0/16",
      "type": "OVNKubernetes"
    },
    "node_drain_grace_period": {
      "unit": "minutes",
      "value": 60
    },
    "nodes": {
      "availability_zones": [
        "us-east-1a"
      ],
      "compute": 2,
      "compute_machine_type": {
        "href": "/api/clusters_mgmt/v1/machine_types/m5.xlarge",
        "id": "m5.xlarge",
        "kind": "MachineTypeLink"
      },
      "compute_root_volume": {
        "aws": {
          "size": 300
        }
      },
      "infra": 2,
      "infra_machine_type": {
        "href": "/api/clusters_mgmt/v1/machine_types/r5.xlarge",
        "id": "r5.xlarge",
        "kind": "MachineTypeLink"
      },
      "master": 3
    },
    "openshift_version": "4.15.12",
    "product": {
      "href": "/api/clusters_mgmt/v1/products/rosa",
      "id": "rosa",
      "kind": "ProductLink"
    },
    "properties": {
      "rosa_cli_version": "1.2.40",
      "rosa_creator_arn": "arn:aws:iam::*************:user/jkeyne"
    },
    "region": {
      "href": "/api/clusters_mgmt/v1/cloud_providers/aws/regions/us-east-1",
      "id": "us-east-1",
      "kind": "CloudRegionLink"
    },
    "state": "ready",
    "status": {
      "configuration_mode": "full",
      "dns_ready": true,
      "kind": "ClusterStatus",
      "limited_support_reason_count": 0,
      "oidc_ready": true,
      "provision_error_code": "",
      "provision_error_message": "",
      "state": "ready"
    },
    "subscription": {
      "href": "/api/accounts_mgmt/v1/subscriptions/2gn3TiilDDdARQw7Rww26saSQql",
      "id": "2gn3TiilDDdARQw7Rww26saSQql",
      "kind": "SubscriptionLink"
    },
    "version": {
      "channel_group": "stable",
      "end_of_life_timestamp": "2025-06-30T00:00:00Z",
      "href": "/api/clusters_mgmt/v1/versions/openshift-v4.15.12",
      "id": "openshift-v4.15.12",
      "kind": "Version",
      "raw_id": "4.15.12"
    }
  }
                                                                                                                                                                                             
  [PANICKED] in [It] - /home/jericho/work/repos/rosa/tests/utils/profilehandler/profile_handler.go:39 @ 05/22/24 16:13:22.002
  << Timeline
                                                                                                                                                                                             
  [PANICKED] Test Panicked
  In [It] at: /home/jericho/work/repos/rosa/tests/utils/profilehandler/profile_handler.go:39 @ 05/22/24 16:13:22.002
                                                                                                                                                                                             
  ENV Variable TEST_PROFILE is empty, please make sure you set the env value
                                                                                                                                                                                             
  Full Stack Trace
    github.com/openshift/rosa/tests/utils/profilehandler.LoadProfileYamlFileByENV()
        /home/jericho/work/repos/rosa/tests/utils/profilehandler/profile_handler.go:39 +0x3c5
    github.com/openshift/rosa/tests/e2e.init.func8.4()
        /home/jericho/work/repos/rosa/tests/e2e/test_rosacli_ingress.go:191 +0x5c
------------------------------
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
                                                                                                                                                                                             
Summarizing 1 Failure:
  [PANICKED!] Edit default ingress [It] can customize ingress controller at install - [id:65798] [day2, feature-ingress, High, day1-post, NonHCPCluster]
  /home/jericho/work/repos/rosa/tests/utils/profilehandler/profile_handler.go:39
                                                                                                                                                                                             
Ran 1 of 79 Specs in 1.578 seconds
FAIL! -- 0 Passed | 1 Failed | 0 Pending | 78 Skipped
--- FAIL: TestROSACLIProvider (1.58s)
FAIL
                                                                                                                                                                                             
Ginkgo ran 1 suite in 2.878350604s
                                                                                                                                                                                             
Test Suite Failed
```

Output with wrong `TEST_PROFILE`:
```
$ TEST_PROFILE=rosa-upgrade-z-stream CLUSTER_ID=2bdae83sg599t6mvhs7f32k99kmfmnhr ginkgo run -focus 65798 tests/e2e/
Running Suite: e2e tests suite - /home/jericho/work/repos/rosa/tests/e2e
========================================================================
Random Seed: 1716408862
                                                                                                                                                                                             
Will run 1 of 79 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
                                                                                                                                                                                             
Ran 0 of 79 Specs in 1.111 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 79 Skipped
PASS
                                                                                                                                                                                             
Ginkgo ran 1 suite in 2.045856232s
Test Suite Passed
```

### CODE REVIEW

File: tests/e2e/test_rosacli_ingress.go [hunk 1]
```diff
@@ -178,6 +179,41 @@ var _ = Describe("Edit default ingress",
 				ingress, _ = defaultIngress(*ingressList)
 				Expect(ingress.LBType).Should(ContainSubstring(ingress.LBType))
 			})
+
+		It("can customize ingress controller at install - [id:65798]",
+			labels.High,
+			labels.Day1Post,
```
Comment (yasun1) [inline tests/e2e/test_rosacli_ingress.go:185]: [Here](https://github.com/openshift/rosa/pull/2093/files#diff-b1d5ebe3991fe8d02791246cbf7cdc7a8eabf417254dd7c3e47c4ae6701acc19R17) defines the cases under this group is `Day2`, is this `Day1Post`? We need to remove the `Day2` from `Description` but add it to each case.

File: tests/e2e/test_rosacli_ingress.go [hunk 2]
```diff
@@ -178,6 +179,41 @@ var _ = Describe("Edit default ingress",
...
+				defaultIngress := func(ingressList rosacli.IngressList) (*rosacli.Ingress, bool) {
+					for _, ingress := range ingressList.Ingresses {
+						if ingress.Default == "yes" {
+							return &ingress, true
+						}
+					}
+					return nil, false
+				}
+
+				ingress, _ := defaultIngress(*ingressList)
```
Comment (yasun1) [inline tests/e2e/test_rosacli_ingress.go:208]: Before call `ingress`, need to do `Expect(ingress).NotTo(BeNil())` or `Expect(existed).To(BeTrue())`

**Comment by yasun1** [comment]
For the job is based on the day-1 in rosa but not prow ci, let us skip running the presubmit jobs.

File: tests/e2e/test_rosacli_ingress.go [hunk 3]
```diff
@@ -178,6 +179,41 @@ var _ = Describe("Edit default ingress",
 				ingress, _ = defaultIngress(*ingressList)
 				Expect(ingress.LBType).Should(ContainSubstring(ingress.LBType))
 			})
+
+		It("can customize ingress controller at install - [id:65798]",
+			labels.High,
+			labels.Day1Post,
+			labels.NonHCPCluster,
+			func() {
+				if isHosted {
```
Comment (radtriste) [inline tests/e2e/test_rosacli_ingress.go:188]: if you place the `NonHCPCluster` label, you don't need to check for isHosted

File: tests/e2e/test_rosacli_ingress.go [hunk 4]
```diff
@@ -178,6 +179,41 @@ var _ = Describe("Edit default ingress",
...
+					Skip("This case is for standard ROSA clusters only")
+				}
+				if !PH.LoadProfileYamlFileByENV().ClusterConfig.IngressCustomized {
```
Comment (radtriste) [inline tests/e2e/test_rosacli_ingress.go:191]: For readability
```suggestion
				profile := PH.LoadProfileYamlFileByENV()
				if !profile.ClusterConfig.IngressCustomized {
```

File: tests/e2e/test_rosacli_ingress.go [hunk 5]
```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/openshift/rosa/tests/utils/common"
 	"github.com/openshift/rosa/tests/utils/config"
 	"github.com/openshift/rosa/tests/utils/exec/rosacli"
+	PH "github.com/openshift/rosa/tests/utils/profilehandler"
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_ingress.go:13]: Do not use capitalized import.

File: tests/e2e/test_rosacli_ingress.go [hunk 6]
```diff
@@ -178,6 +179,39 @@ var _ = Describe("Edit default ingress",
...
+					for _, ingress := range ingressList.Ingresses {
+						if ingress.Default == "yes" {
+							return &ingress, true
+						}
+					}
+					return nil, false
+				}
+
+				ingress, _ := defaultIngress(*ingressList)
+				Expect(ingress.ExcludeNamespace).Should(ContainSubstring("test-ns1"))
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_ingress.go:207]: There shouldn't be hardcode here. You have to read the config file and verify it is expected to equal the value in config file

Comment (jerichokeyne) [reply tests/e2e/test_rosacli_ingress.go]: @xueli181114 I believe I fixed it the way you recommended, can you take another look?

**Comment by radtriste** [comment]
@xueli181114 Can you have a look to the changes ?

**Comment by xueli181114** [comment]
@jerichokeyne  This PR needs rebase

**Comment by jerichokeyne** [comment]
@xueli181114 I've rebased it now. Could you take another look at the comment you left before, and see if I implemented the changes you wanted

**Pull request merged**